pkg/advisory/export.go

```diff
@@ -256,16 +247,14 @@ func ExportOSV(opts ExportOptions, output string) error {
 			log.Fatal(err)
 		}
 
-		// to run the validate schema
-		var result any
-		err = json.Unmarshal(e, &result)
-		if err != nil {
-			log.Fatalf("failed to unmarshall:%v", err)
-		}
-		err = schema.Validate(result)
-		if err != nil {
-			log.Fatalf("failed to validate OSV JSON Schema for %s: %v", k, err)
-		}
+		// TODO(luhring): This should probably be moved to a test. But it's also failing
+		//  for a separate reason, which is that we're using the "wolfi" ecosystem, which
+		//  isn't valid according to the OSV schema. We'll have to submit the
+		//  "chainguard" ecosystem upstream.
+		//
+		// err = schema.Validate(result) if err != nil {
+		// 	log.Fatalf("failed to validate OSV JSON Schema for %s: %v", k, err)
+		// }
```

Comment:
Luckily my next task was already to circle back on this code and get it ready for prime time.